gtfsmerger/__init__.py

```diff
@@ -2,6 +2,21 @@
 from gtfsmerger.gtfs import GTFS
 
 
+def post_process(func):
+    def wrapper(self, gtfs_obj):
+        merged = func(self, gtfs_obj)
+        if 'calendar_dates' in merged:
+            merged['calendar_dates'].loc[:, 'date'] = pd.to_datetime(merged[
+                'calendar_dates'].loc[:, 'date'].astype(str))
```

Comment:
Pandas see data columns as int64 originally, if we convert that to datetime, pandas will see it as tick time. By converting that to string first, pandas is able to recognize it as timestamp